src/actions/Upload.ts

```diff
@@ -15,6 +15,8 @@ export interface IUploadParams {
   releaseNotes?: IReleaseNotes[]
   /** An array that specifies the paths to the expansion files (OBBs) for this release. */
   obbs?: string[]
+  /** An array that specifies the paths to the deobfuscation files for this release. */
+  deobfuscation?: string
```

Comment:
Thanks for documenting it! 

Is there only ever one deobfuscation file?

Reply:
Isn't it makes sense? One Apk - One deobfuscation

Reply:
Just wanted to clarify since I honestly did not know for sure :)